canvas_modules/common-canvas/package.json

```diff
@@ -42,6 +42,7 @@
     "react-resize-detector": "5.2.0",
     "react-virtualized": "9.22.2",
     "redux": "4.0.5",
+    "sass": "1.32.2",
```

Comment:
This should be a devDependency since it's only needed at build time.

Reply:
fixed, thanks!